From 414fd06af6eee2e9d9b048415bd13f7d09a13d62 Mon Sep 17 00:00:00 2001
From: Ken Cunningham <kencu@macports.org>
Date: Thu, 3 May 2018 12:53:27 -0700
Subject: [PATCH] ycbcr: assembly fixes for old gcc-as in Tiger

minor changes in assembly that are friendly to the older
gcc-as in Tiger
---
 gfx/ycbcr/yuv_row_posix.cpp | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/gfx/ycbcr/yuv_row_posix.cpp b/gfx/ycbcr/yuv_row_posix.cpp
index 818ec5d11..a3494cf31 100644
--- a/gfx/ycbcr/yuv_row_posix.cpp
+++ b/gfx/ycbcr/yuv_row_posix.cpp
@@ -272,8 +272,7 @@ void FastConvertYUVToRGB32Row_SSE(const uint8* y_buf,
                                   int width);
   asm(
   ".text\n"
-  ".global FastConvertYUVToRGB32Row_SSE\n"
-  ".type FastConvertYUVToRGB32Row_SSE, @function\n"
+  ".globl FastConvertYUVToRGB32Row_SSE\n"
 "FastConvertYUVToRGB32Row_SSE:\n"
   "pusha\n"
   "mov    0x24(%esp),%edx\n"
@@ -350,8 +349,7 @@ void ScaleYUVToRGB32Row_SSE(const uint8* y_buf,
                             int source_dx);
   asm(
   ".text\n"
-  ".global ScaleYUVToRGB32Row_SSE\n"
-  ".type ScaleYUVToRGB32Row_SSE, @function\n"
+  ".globl ScaleYUVToRGB32Row_SSE\n"
 "ScaleYUVToRGB32Row_SSE:\n"
   "pusha\n"
   "mov    0x24(%esp),%edx\n"
@@ -444,8 +442,7 @@ void LinearScaleYUVToRGB32Row_SSE(const uint8* y_buf,
                                   int source_dx);
   asm(
   ".text\n"
-  ".global LinearScaleYUVToRGB32Row_SSE\n"
-  ".type LinearScaleYUVToRGB32Row_SSE, @function\n"
+  ".globl LinearScaleYUVToRGB32Row_SSE\n"
 "LinearScaleYUVToRGB32Row_SSE:\n"
   "pusha\n"
   "mov    0x24(%esp),%edx\n"
